src/containers/TaskRuns/TaskRuns.js

```diff
@@ -76,9 +77,12 @@ export /* istanbul ignore next */ class TaskRunsContainer extends Component {
   }
 
   componentDidUpdate(prevProps) {
-    const { match } = this.props;
+    const { match, namespace } = this.props;
     const { taskName } = match.params;
-    if (taskName !== prevProps.match.params.taskName) {
+    if (
+      taskName !== prevProps.match.params.taskName ||
+      namespace !== prevProps.namespace
```

Comment:
There isn't any generic way to handle these namespace checks ?

Reply:
Discussed offline, this is being tracked by https://github.com/tektoncd/dashboard/issues/137